app/javascript/dashboard/components/widgets/TableFooter.vue

```diff
@@ -12,39 +12,41 @@
         v-if="totalCount"
         class="primary button-group pagination-button-group"
       >
-        <button
-          class="button small goto-first"
+        <woot-button
+          size="small"
+          class-names="goto-first"
           :class="firstPageButtonClass"
```

Comment:
This `firstPageButtonClass` computed method is to get `disabled` class. But we have `is-disabled` prop for button. Please use the props.